### ISSUE TITLE
v2: Menu content vs Gui content

### ISSUE THREAD

**Issue by Ragnar-F** [Lexikos/AutoHotkey_L-Docs#213]
First of all: Nice changes regarding the new menu object. Thanks.

But when it comes to documentation, I notice a discrepancy in the content separation of **commands/Menu.htm** compared to the content separation of **commands/Gui.htm**.

Currently, **commands/Menu.htm** has been split to **objects/Menu.htm** and **commands/MenuCreate.htm**, while **commands/Gui.htm** has been split to **objects/Gui.htm**, **objects/GuiControl.htm** and **commands/GuiCreate.htm**.

So far so good, but:

- **objects/Menu.htm** contains the documentation about its object members, win32 menus, remarks, related, examples
- **commands/MenuCreate.htm** contains only the documentation about itself

Compared to:

- **objects/Gui.htm** and **objects/GuiControl.htm** contain only the documentation about their object members
- **commands/GuiCreate.htm** contains the documentation about itself, keyboard navigation, window appearance, general remarks, related, examples

We should only choose one direction. I think that a better overview is given if **commands/MenuCreate.htm** contains the documentation about win32 menus, remarks, related and examples, while **objects/Menu.htm** focuses only on its object members, as it is already the case with the other objects/ pages.

**Comment by Lexikos** [comment]
I chose one direction.  As a matter of fact, I even [pointed you in that direction](https://github.com/Lexikos/AutoHotkey_L-Docs/pull/175#issuecomment-292817699):

> "GuiCreate" is very specific, and if it remains its own page, should only be about that function.

I did not care that you did not follow my suggestion, as I saw the matter as inconsequential.

I chose this way because the majority of the content from commands/Menu.htm was about sub-commands/methods, making it more or less a topic page about Menu.  By contrast, MenuCreate() is an extremely simple function which hardly even needs to be documented.  It might even be better placed on the Menu object page, as it is merely the Menu object "constructor".

There are advantages to having all of the information on one page, and none that I can see to having half of it on the MenuCreate page.
